Maquetado/index.html

```diff
@@ -12,6 +12,7 @@
 <body>
   <div class='bookContainer'>
     <img src='./resources/img/book-cover.png' class='bookImage' />
+    <img src='./resources/img/badge.png' class='bookBadge' />
```

Comment:
This is the perfect case where you could add this image as a pseudoelement.
This is an example: https://impossiblue.github.io/log/170326/index.html